commands.json

```diff
@@ -1,11 +1,12 @@
 [
   {
-    "command_name": [
-      "help",
+    "command_name": [ // the commands name, you can have multiple
+      "help", // will respond to (prefix)help
       "h",
       "?"
     ],
-    "call": "This is your help command. (fill in in commands.json)"
+    "call": "This is your help command. (fill in in commands.json)" 
+    // What the bot responds with, can only be a message as of now, direct message/command support soon (maybe)
```

Comment:
I think comments in JSON are not allowed, we should put this somewhere in `readme.md`

Reply:
Fair enough, i think it semi supports comments but it looks like "_comment": "bla bla" I will put the info on how to make them in readme